topi/python/topi/x86/conv2d_avx_1x1.py

```diff
@@ -2,14 +2,16 @@
 """1x1 Conv2D schedule on for Intel CPU"""
 from __future__ import absolute_import as _abs
 from collections import namedtuple
+import multiprocessing
 import tvm
 
 from ..util import get_const_tuple
 from ..nn.conv2d import _get_schedule, _get_workload
 from ..nn.util import infer_pad, infer_stride
 from ..nn.pad import pad
 
-AVXConv1x1Fwd = namedtuple('AVXConv1x1Fwd', ['ic_bn', 'oc_bn', 'oh_factor', 'ow_factor'])
+AVXConv1x1Fwd = namedtuple('AVXConv1x1Fwd',
+                           ['ic_bn', 'oc_bn', 'oh_factor', 'ow_factor', 'parallel_chunk'])
```

Comment:
It looks like parallel_chunk is purely decided from other workload and schedule parameters together with multiprocessing.cpu_count(). I suggest we don't add it as another schedule parameter, since it can't be arbitrarily selected.